include/core/ui_edge_insets.hpp

```
@@ -11,11 +11,17 @@ struct UIEdgeInsets {
   float bottom = 0.0f;
   float right = 0.0f;
 
-  static UIEdgeInsets horizontal(float horizontalSpace) { return UIEdgeInsets{.left = horizontalSpace, .right = horizontalSpace}; }
+  static UIEdgeInsets horizontal(float horizontalSpace) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)